server/eclipse-project/src/main/java/us/freeandfair/corla/query/CVRAuditInfoQueries.java

```diff
@@ -81,6 +102,127 @@ private CVRAuditInfoQueries() {
   }
   
   /**
+   * Obtain the sublist of CVRAuditInfo objects for the specified 
```

Comment:
It would be nice if we, in a more human-friendly form, succinctly summarized the intended strategy being used for the queries constructed in these methods.